src/Microsoft.Diagnostics.Runtime/Desktop/heap.cs

```diff
@@ -1667,6 +1667,10 @@ public override ClrType GetObjectType(Address objRef)
             return type;
```

Comment:
Changes in this file should be reverted.  You should implement GetTypeByMethodTable for .Net Native instead.